debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorTask.java

```diff
@@ -70,20 +72,35 @@ public ChangeEventSourceCoordinator start(Configuration config) {
             throw new ConnectException("Unable to load snapshotter, if using custom snapshot mode, double check your settings");
         }
 
+        heartbeatConnection = new PostgresConnection(connectorConfig.jdbcConfig());
+        final Charset databaseCharset = heartbeatConnection.getDatabaseCharset();
+
+        final Function<TypeRegistry, PostgresValueConverter> valueConverterBuilder = (typeRegistry) -> new PostgresValueConverter(
```

Comment:
I wasn't sure whether it would make most sense to call this 'builder'/'generator'/etc. but I've just wrapped up the configuration for the `PostgresValueConverter` into a function which will construct a `PostgresValueConverter` for a given `TypeRegistry`